packages/lab/src/link-card/LinkCard.css

```diff
@@ -0,0 +1,97 @@
+  border-style: var(--saltCard-borderStyle, var(--salt-container-borderStyle));
+  border-radius: var(--saltCard-borderRadius, 0);
+  display: block;
+  padding: var(--saltCard-padding, var(--salt-spacing-200));
+  position: relative;
+  text-decoration: none;
+  transition: box-shadow var(--salt-duration-instant) cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+/* Styles applied to LinkCard when variant="primary" */
+.saltLinkCard-primary {
+  background: var(--saltCard-background, var(--salt-container-primary-background));
+  border-color: var(--salt-container-primary-borderColor);
+  --card-accentColor: var(--saltCard-accentColor, var(--salt-container-primary-borderColor));
+}
+
+/* Styles applied to LinkCard when variant="secondary" */
+.saltLinkCard-secondary {
+  background: var(--saltCard-background, var(--salt-container-secondary-background));
+  border-color: var(--salt-container-secondary-borderColor);
+  --card-accentColor: var(--saltCard-accentColor, var(--salt-container-secondary-borderColor));
+}
+
+/* Styles applied to create accent */
+.saltLinkCard-accent::after {
+  content: "";
+  position: absolute;
+  background-color: var(--card-accentColor);
+}
+
+/* Styles applied to LinkCard if `accent="bottom"` (default) */
+.saltLinkCard-accentBottom::after {
+  left: calc(-1 * var(--salt-size-border));
+  bottom: calc(-1 * var(--salt-size-border));
+  height: var(--salt-size-accent);
+  width: calc(100% + calc(2 * var(--salt-size-border)));
+}
+
+/* Styles applied to LinkCard if `accent="left"` */
+.saltLinkCard-accentLeft::after {
+  left: calc(-1 * var(--salt-size-border));
+  top: calc(-1 * var(--salt-size-border));
+  height: calc(100% + calc(2 * var(--salt-size-border)));
+  width: var(--salt-size-accent);
+}
+
+/* Styles applied to LinkCard if `accent="top"` */
+.saltLinkCard-accentTop::after {
+  left: calc(-1 * var(--salt-size-border));
+  top: calc(-1 * var(--salt-size-border));
+  height: var(--salt-size-accent);
+  width: calc(100% + calc(2 * var(--salt-size-border)));
+}
+
+/* Styles applied to LinkCard if `accent="right"` */
+.saltLinkCard-accentRight::after {
+  right: calc(-1 * var(--salt-size-border));
+  top: calc(-1 * var(--salt-size-border));
+  height: calc(100% + calc(2 * var(--salt-size-border)));
+  width: var(--salt-size-accent);
+}
+
+/* Styles applied to LinkCard on focus */
+a:focus-visible .saltLinkCard,
+.saltLinkCard:focus-visible {
+  cursor: var(--saltCard-Link-cursor-focus, var(--salt-selectable-cursor-hover));
+  box-shadow: var(--saltCard-Link-shadow-focus, var(--salt-overlayable-shadow-hover));
+  color: var(--saltCard-color-focus, var(--salt-content-primary-foreground));
+  outline-color: var(--salt-focused-outlineColor);
+  outline-style: var(--salt-focused-outlineStyle);
+  outline-width: var(--salt-focused-outlineWidth);
+  outline-offset: var(--salt-focused-outlineOffset);
+  --card-accentColor: var(--saltCard-accentColor-hover, var(--salt-selectable-foreground-hover));
```

Comment:
Can we remove the --saltCard-* variables here. Our current rule is to now create them specific to a state/variant.

Reply:
I see, but I'm not sure then how we want the user to be able to override the accent color on hover. Would they use --card-accentColor?

Reply:
They can use a selector at the variable without state